xml/cap_depl_azure.xml

```diff
@@ -406,59 +409,70 @@ suse       https://kubernetes-charts.suse.com/
 
  </sect1>
  <sect1 xml:id="sec.cap.cap-on-azure">
-  <title>Deploy &productname; with a Load Balancer</title>
-
-  <para>
-   In &productname;, load balancing is enabled by setting the
-   <literal>services.loadbalanced</literal> parameter to
-   <literal>true</literal> and specifying a fully qualified domain name (FQDN)
-   for the <literal>DOMAIN</literal> parameter in your configuration file. When
-   <literal>services.loadbalanced</literal> is set to <literal>true</literal>,
-   the
-   <link xlink:href="https://kubernetes.io/docs/concepts/services-networking/#publishing-services-service-types">ServiceType</link>
-   of public-facing services will change to
-   <link xlink:href="https://kubernetes.io/docs/concepts/services-networking/#loadbalancer">Type
-   LoadBalancer</link> and a load balancer will be provisioned for the service.
-   In &azureks;, this provisions a
-   <link xlink:href="https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-overview#skus">Basic
-   Load Balancer</link>.
-  </para>
-
+     <title>Deploying &productname; with a Load Balancer</title>
   <para>
-   The following configuration examples show how to configure the required
-   storage class, and provide a complete example configuration for the
-   deployment.
+      This section describes how to deploy &productname; with a basic
+      AKS load balancer, how to set up the required storage volume, and how
+      to configure your DNS records.
   </para>
+<sect2 xml:id="sec.cap.aks-storage">
+    <title>Creating the Required Storage Volume</title>
   <para>
-   AKS clusters contain two built-in storage classes (see
+   AKS clusters contain two built-in storage classes, the <literal>default</literal> 
+   storage class and the <literal>managed-premium</literal>
+   storage class. Deploying &cap; on AKS requires provisioning an Azure disk 
+   with the <literal>managed-premium</literal> storage class. Follow the steps
+   at
    <link xlink:href="https://docs.microsoft.com/en-us/azure/aks/azure-disks-dynamic-pv">
-       Dynamically create and use a persistent volume with Azure disks in AKS</link>,
-   the <literal>default</literal> storage class and the <literal>managed-premium</literal>
-   storage class. Deploying &cap; on AKS requires provisioning an Azure disk with the
-   <literal>managed-premium</literal> storage class. In your deployment 
-   configuration file ensure that <literal>managed-premium</literal> has been 
-   specified as the value for <literal>kube.storage_class.persistent</literal>, 
-   like this example:
+       Dynamically create and use a persistent volume with Azure disks in AKS</link>)
```

Comment:
Still not quite sure I understand why the steps in the link need to be followed. From my understanding, the needed persistent volume(s) and persistent volume claim(s) are created by CAP as long as a storage class name is passed to kube.storage_class.persistent in the values.yaml file.

But the storage section needs to be updated in any case. I looked at what QA is doing and they're creating their own storage class and using that rather than the built-in storage class.

Reply:
I had to do this for a successful deployment. UAA failed with an unhelpful error message if I didn't.

Reply:
I'll go ahead and merge, but will make a separate commit to update the content to match the steps QA uses for storage